tests/typecheck/contrib/sitemaps/test_generic_sitemap.yml

```diff
@@ -0,0 +1,62 @@
+-   case: test_items_custom_model
+    main: |
+        from django.contrib.sitemaps import Sitemap
+        from django.db.models import QuerySet
+        from django.urls import reverse
+        from myapp.models import Offer
+
+        class OfferSitemap(Sitemap[Offer]):
+            priority = 1
+            changefreq = "always"
+
+            def items(self) -> QuerySet[Offer]:
+                return Offer.objects.all()
+
+            def location(self, item: Offer) -> str:
+                return reverse(
+                    "myapp:detail-offer",
+                    kwargs={
+                        "provider_name": item.provider,
+                        "offer_name": item.trc, # E: "Offer" has no attribute "trc"
+                    },
+                )
+
+        class WrongOfferSitemap(Sitemap[Offer]):
```

Comment:
One last question: what about `GenericSiteMap`? Should we test it?

Reply:
Yeah I was unsure about this one. I don't know how people are generally using it. 

Maybe I should add a case close to [the documentation exemple](https://docs.djangoproject.com/en/4.1/ref/contrib/sitemaps/#django.contrib.sitemaps.GenericSitemap) to make sure everything pass ?

Reply:
👍